pupy/modules/cat.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: UTF8 -*-
+from pupylib.PupyModule import *
+from pupylib.utils.rpyc_utils import redirected_stdio
+
+__class_name__="cat"
+
+@config(cat="admin")
+class cat(PupyModule):
+	""" remove a file or a directory """
```

Comment:
I believe this is a copy-pasta. "show contents of a file" or anything appropriate.